### PR TITLE
Move Support page under Project and move Kuberentes Compatibility to Operate

### PR DESCRIPTION
**Pull request by zroubalik**
Signed-off-by: Zbynek Roubalik <zroubalik@gmail.com>

_Provide a description of what has been changed_

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Relates to https://github.com/kedacore/keda-docs/pull/910

### CODE REVIEW

File: content/support.md [hunk 1]
```diff
@@ -14,12 +14,3 @@ Learn more in our [support policy](https://github.com/kedacore/governance/blob/m
 Here's an overview of all vendors that provide KEDA as part of their offering/product and provide support for it: 
 
 {{< support >}}
-
-## Kubernetes support
```
Comment (tomkerkhove) [inline content/support.md:18]: That's fine for me, but we might want to add a link in the OSS support section to /latest/operate/cluster

Comment (zroubalik) [reply content/support.md:18]: Fixed

File: content/support.md [hunk 2]
```diff
@@ -10,16 +10,9 @@ Want to contribute a feature or fix? We are more than happy to review requests a
 
 Learn more in our [support policy](https://github.com/kedacore/governance/blob/main/SUPPORT.md).
 
+Kubernetes compatibility is described in the [documentation](https://keda.sh/docs/2.8/operate/cluster/#kubernetes-compatibility).
```
Comment (tomkerkhove) [inline content/support.md:13]:
```suggestion
Kubernetes compatibility is described in the [documentation](https://keda.sh/docs/latest/operate/cluster/#kubernetes-compatibility).
```

**Review by tomkerkhove — APPROVED** [review]
LGTM

**Comment by tomkerkhove** [comment]
GitHub and its stupid "sign off commit" bug in code suggestions -.-

Merging as is

**Comment by zroubalik** [comment]
@tomkerkhove omg, good catch

**Comment by tomkerkhove** [comment]
If it was a good catch, then I would have noticed before merging.

No problem!

**Pull request merged**